hyperspy/_components/gaussian2d.py

```diff
@@ -151,3 +153,8 @@ def rotation_major_axis(self):
             return self.rotation_angle_wrapped
         else:
             return self.rotation_angle_wrapped - np.pi / 2
+
+    @property
+    def height(self):
```

Comment:
Add a `setter` like in the 1D-Gaussian https://github.com/hyperspy/hyperspy/blob/48bf6cb51786ddc8597c969dd46286d1117f40df/hyperspy/_components/gaussian.py#L200